README.rst

```diff
@@ -173,21 +173,10 @@ The analysis workflow is simple and consists of two above-mentioned stages:
 There is a serial workflow that does the steps sequentially, but this demo
 represents a better use case for workflow tools capable of parallel execution.
 
-We shall use the `CWL <http://www.commonwl.org/v1.0/>`_ workflow specification
+We shall use the `Snakemake <https://snakemake.readthedocs.io/en/stable/>`_ workflow specification
```

Comment:
Since we'll remove `reana-debug.yaml` serial workflow, we can remove also phrases like "There is a serial workflow..."

The paragraph could say:

- The steps processing collision data and simulated data can be run in parallel. We shall use the Snakemake workflow specification to express the computational workflow  by means of the following `Snakefile`:

Reply:
BTW please modify the ASCII art diagram because after START we have a "scram" step that produces the code necessary for these analysis steps. 

In this way the diagram would better correspond to the Snakemake content.

Reply:
> after START we have a "scram" step that produces the code necessary for these analysis steps.

This makes the total number of stages of the analysis 3, and not 2 anymore, correct?
If so, should we amend the description of the analysis code to refer to three stages (and not only "the first" and "the second"), and describe the SCRAM step too?